PSDesiredStateConfiguration.md

```diff
@@ -0,0 +1,106 @@
+## Getting Started
+
+**VMware.PSDesiredStateConfiguration** provides a new way to compile and execute **DSC Configurations** on **Windows**, **Linux** and **MacOS** that does not require the use of **DSC LCM** or the typical MOF format and without altering the existing **Configuration** syntax.  Instead the cmdlets of this module create and work with regular powershell objects. This module is also designed to work with the existing **DSC Resources** in **VMware.vSphereDSC**.  
+
+## Requirements
+
+The following table describes the required dependencies for running VMware.vSphereDSC Resources.
+
+ **Required dependency**   | **Minimum version**
+-------------------------- | -------------------
+`PowerShell`               | 5.1 (Core is supported)
+
+1. Copy the VMware.PSDesiredStateConfiguration Module to one of the system PowerShell module directories. For more information on installing PowerShell Modules, please visit [Installing a PowerShell Module](https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershell-7).
+2. In PowerShell import the VMware.PSDesiredStateConfiguration Module:
+   ```
+    Import-Module -Name 'VMware.PSDesiredStateConfiguration'
+   ```
+
+   To check if the module was successfully installed:
+   ```
+    Get-Module -Name 'VMware.PSDesiredStateConfiguration'
+   ```
+This module also has additional requirements depending on which version of powershell you wish to use it with.
+
+#### Powershell core requirements
+---
+By using **Powershell Core** this module depends on the Invoke-DscResource cmdlet and it must be enabled with the following command.
```

Comment:
Before that state the following. 

`VMware.PSDesiredStateConfiguration` uses the `Invoke-DscResource` cmdlet from `PSDesiredStateConfiguration` module to process individual DSC Resources. In PowerShell 7.0 `Invoke-DscResource` cmdlet is experimental feature.